assets/src/components/d3/createHistogram.js

```diff
@@ -34,7 +34,7 @@ function createHistogram ({ data, width, height, el, tip, xAxisLabel, yAxisLabel
 
   bar.append('text')
     .attr('x', d => x((d.x1 + d.x0) / 2))
-    .attr('y', d => y(d.length) + 14)
+    .attr('y', d => y(d.length) + margin.top-5)
```

Comment:
added the MyGrade text top of the line as to make it stand out for student to see where his grade in the distribution